GL/texture.c

```diff
@@ -139,6 +139,41 @@ static void GPUTextureTwiddle8PPP(void* src, void* dst, uint32_t w, uint32_t h)
     }
 }
 
+static void GPUTextureTwiddle4PPP(void* src, void* dst, uint32_t w, uint32_t h) {
+    uint32_t x, y, yout, min, mask;
+
+    min = MIN(w, h);
+    mask = min - 1;
+
+	uint8 * pixels;
+    uint16 * vtex;
+    pixels = (uint8 *) src;
+    vtex = (uint16*)dst;
+
+    for (y=0; y<h; y += 2) {
+        yout = y;
+        for (x=0; x<w; x += 2) {
+
+                //Ozzy note: if x alignment is wrong just use version commented instead (possible error)
```

Comment:
Any more information on this? I think we need to resolve this (and remove the #if) before we merge. Is there an edge case bug?

Reply:
well, the usual 4bpp madness with interlaced pixels [0,1] or [1,0] ..  maybe i've done it in the wrong order ? not sure about it ^^ 

make your choice :)